experiments/landlords/parent_company_search/How to Find Building Owners.ipynb

```diff
@@ -0,0 +1,675 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Finding Building Owners\n",
+    "\n",
+    "The problem: the City of Seattle does not provide us with the owners of buildings in the carbon benchmarking dataset. Without this data, we can't determine which companies have the largest environmental impact on Seattle. Moreover, the companies listed as building owners in the King County property database are often shell companies. There is no database where you can look up the parent company for a shell company. There is also no reliable way to find a parent company's shell companies (though this assumes you're looking for buildings owned by one particular company).\n",
```

Comment:
`Research challenge: We have a dataset of large buildings (>20k sq ft) and their carbon footprint from the [City of Seattle's Office of Sustainability and the Environment](https://www.seattle.gov/environment/climate-change/buildings-and-energy/energy-benchmarking). This means that we can group those buildings by owner and determine the carbon footprint of each owner. However, [insert rest of the original explanation]`